packages/launchpad/src/components/topnav/TopNav.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <TopNavList v-if="versionList">
+    <template #heading="{ open }">
+      <i-cy-box_x16
+        class="group-hocus:icon-dark-indigo-500 group-hocus:icon-light-indigo-50 h-16px w-16px"
+        :class="open ? 'icon-dark-indigo-500 icon-light-indigo-50' : 'icon-dark-gray-500 icon-light-gray-100'"
+      />
+      <span>v{{ versionList[0].version }}</span>
+    </template>
+    <TopNavListItem
+      v-for="(item, index) in versionList"
+      :key="item.version"
+      :selectable="!!index"
+      class="px-4 py-2 min-w-240px"
+      :class="index ? '' : 'bg-jade-50'"
+    >
+      <div class="whitespace-nowrap">
+        <a
+          :href="`${releasesUrl}/tag/v${item.version}`"
+          :class="index ? '' : 'text-jade-600'"
+          class="font-semibold"
+          target="_blank"
+        >{{ item.version }}</a>
+        <br>
+        <span class="text-gray-600 text-14px">Released {{ item.released }}</span>
+      </div>
+      <template
+        v-if="!index"
+        #suffix
+      >
+        <i-cy-circle-check_x24
+          class="icon-dark-jade-100 icon-light-jade-500 w-24px h-24px"
+        />
+      </template>
+    </TopNavListItem>
+    <TopNavListItem class="p-4 text-center bg-gray-50">
+      <a
+        :href="releasesUrl"
+        target="_blank"
+        class="block w-full py-2 border-gray-100 text-14px whitespace-nowrap border-rounded border-1 hover:no-underline hover:border-gray-200"
+      >See all releases</a>
+    </TopNavListItem>
+  </TopNavList>
+
+  <TopNavList v-if="props.gql?.selectedBrowser && props.showBrowsers">
```

Comment:
Props are automatically unpacked. Because `gql` the prop shares the same name as the import, we put it under "props", but that's actually a pretty weird pattern.

```suggestion
  <TopNavList v-if="props.gql?.selectedBrowser && showBrowsers">
```